third_party/vault/Makefile

```diff
@@ -17,7 +17,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 		--set "vault.global.openshift=$(WITH_OPENSHIFT)" \
                 --set "vault.injector.enabled=false" \
                 --set "vault.server.dev.enabled=true" \
-		--values $(VAULT_VALUES_FILE) \
+		--values $(ROOT_DIR)/$(VAULT_VALUES_FILE) \
```

Comment:
The changes in this file make it impossible to provide a values file from a directory not under the project root. I'm unsure what's the benefit of this change is.

Reply:
this original one, it's not right:
`export VAULT_VALUES_FILE=../../$(ROOT_DIR)charts/vault/env/dev/local-multi-cluster.yaml`
1. in shell, we can not use $()
2. we always use the yaml files start from `charts` folder
3. if you need to use the whole path of the yaml file, you need to append a 'ROOT_DIR', just like this fix

Reply:
I understand the changes in the other file. For this file I understand from your comment that without this change a relative path in VAULT_VALUES_FILE is relative to `third_party/vault` and that's not what we want. So I'm ok with this change. Thanks.